### PR TITLE
Feat/config component

### PR DESCRIPTION
**Pull request by Crash--**
- Add a config.json file 
- Display the promotion banner if activated

### CODE REVIEW

**Comment by Crash--** [comment]
[(𝐋) Custo Drive pour Orange 🍊](https://trello.com/c/ML4nmtCW/1762-%F0%9D%90%8B-custo-drive-pour-orange-%F0%9F%8D%8A)

File: src/components/pushClient/Banner.jsx [hunk 1]
```diff
@@ -23,24 +24,27 @@ class BannerClient extends Component {
...
+    this.setState({ mustShow: false })
     track(element)
   }
 
   render() {
+    if (Config.banner.isActivated !== true) return null
     const { t } = this.props
-    if (!this.state.mustShow) return null
+
+    if (this.state.mustShow !== true) return null
```
Comment (cedricmessiant) [inline src/components/pushClient/Banner.jsx]: why do you prefer this `this.state.mustShow !== true`?

File: src/viewer/NoViewer/CallToAction.jsx [hunk 2]
```diff
@@ -8,13 +8,15 @@ import {
...
 import styles from '../styles'
+import Config from '../../drive/config/config.json'
 
 export default class CallToAction extends Component {
   state = {
     mustShow: false
   }
 
   async componentDidMount() {
+    if (Config.banner.isActivated !== true) return
```
Comment (cedricmessiant) [inline src/viewer/NoViewer/CallToAction.jsx]: is this `banner` or `desktopPromo`?

File: src/components/pushClient/Banner.jsx (hunk 1 above)
Comment (Crash--) [reply src/components/pushClient/Banner.jsx]: This modification should not be there

File: src/viewer/NoViewer/CallToAction.jsx (hunk 2 above)
Comment (Crash--) [reply src/viewer/NoViewer/CallToAction.jsx]: renamed to promoteDesktop : true / falme

**Pull request merged**